Simplenote/Classes/CSSearchable+Helpers.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CSSearchableItem+Helpers.swift
+//  Simplenote
+//
+//  Created by Michal Kosmowski on 25/11/2016.
+//  Copyright © 2016 Automattic. All rights reserved.
+//
+
+import Foundation
+import CoreSpotlight
+import MobileCoreServices
+
+extension CSSearchableItemAttributeSet {
+
+    convenience init(note: Note) {
+        self.init(itemContentType: kUTTypeData as String)
+        title = note.titlePreview
+        let index = note.preview.index(note.preview.startIndex, offsetBy: note.titlePreview.utf16.count)
+        contentDescription = note.preview.substring(from: index)
+    }
+    
+}
+
+extension CSSearchableItem {
+
+    convenience init(note: Note) {
+        let set = CSSearchableItemAttributeSet(note: note)
+        self.init(uniqueIdentifier: note.simperiumKey, domainIdentifier:"notes", attributeSet: set)
+    }
+
+}
+
+extension CSSearchableIndex {
+
+    func indexSearchableNote(_ note: Note) {
+        let item = CSSearchableItem(note: note)
+        CSSearchableIndex.default().indexSearchableItems([item]) { error in
```

Comment:
Since this method is always called as `[CSSearchableIndex defaultSearchableIndex]`, perhaps we could nuke  `CSSearchableIndex.default()`, and call `indexSearchableItems` directly?